sharktank/tests/ops/pipeline_parallelized_test.py

```diff
@@ -294,39 +141,34 @@ def testCatNonSplitDimPinned(self):
             unsharded_result, count=shard_count, dim=shard_dim
         )
         expected_result = expected_result.clone(
-            devices=tuple(4 + i for i in range(shard_count)), pinned=True
+            devices=tuple(4 + i for i in range(shard_count))
```

Comment:
Usually I recommend putting a `variable = 4` above then using the variable so the value has a defined explanation.

Reply:
Will fix when I re-enable. What I'm trying to do is ensure that there isn't overlap between the specified devices and the default devices. I am changing it to `tuple(range(shard_count, 2 * shard_count))` to guarantee the lack of overlap and I'll add a comment.